imageroot/bin/module-cleanup-state

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+rm -vf restore/*.sql
```

Comment:
As the module-dump-state creates the subdir, the cleanup must remove it

```suggestion
rm -rvf restore/
```

Reply:
done